global_variable_interface.m

```diff
@@ -0,0 +1,10 @@
+// Global Variable Implementation
+// 
+//
+// IDECodeSnippetCompletionPrefix: vokGlobalVariableImplementation
+// IDECodeSnippetCompletionScopes: [TopLevel]
+// IDECodeSnippetIdentifier: DCB5228E-34F4-4453-BF27-0A022BFF403E
+// IDECodeSnippetLanguage: Xcode.SourceCodeLanguage.Objective-C
+// IDECodeSnippetUserSnippet: 1
+// IDECodeSnippetVersion: 2
+<#Type#> *const <#XXXGlobalVariableName#> = <#Value#>;
```

Comment:
On second thought I will update the documentation on this one to say how this is to be used for object variables.  And will add another snippet for non object variables in a following PR to keep this git history as clean as possible if that is ok.